sdcm/tester.py

```diff
@@ -1247,6 +1247,73 @@ def _check_build_started():
     def rows_to_list(rows):
         return [list(row) for row in rows]
 
+    def create_table_as_another_table(self, keyspace, create_from_name, new_table_name):
```

Comment:
better API:
```
copy_table(self, src, dest):
    " src: "keyspace.table" syntax
       dest: "keyspace.table" syntax 
```
same for MV: `copy_mv_table(self, src, dest)`

Reply:
alternatively `copy_table(self, src_keyspace, src_table,  dest_keyspace, dest_table):`

Reply:
Done